### ISSUE TITLE
Will there be a 1.19.2 version of this mod?

### ISSUE THREAD

**Issue by MrTigerTL** [Giant-Salted-Fish/Key-Binding-Patch#10]
Would be appreciated. Thanks!

**Comment by Giant-Salted-Fish** [comment]
No, I think. Barely have time now to support a version that may not have a large player base. An this mod suffers from compatibility problems, and may not work as you expected even after the adaptation.

Feel free to find someone else who has time to transfer this mod to 1.19.2 if you urgent to use it for your game.
